tests/text/generate/test_integration_transfomers.py

```diff
@@ -114,7 +114,7 @@ def test_transformers_integration_choice():
 
 def test_transformers_integration_with_pad_token():
     model_name = "hf-internal-testing/tiny-random-XLMRobertaXLForCausalLM"
-    model = models.transformers(model_name, device="cpu")
+    model = models.transformers(model_name, device="meta")
```

Comment:
This was necessary in order to get around some "meta device" issues related to `accelerate`.